common/schemas.py

```diff
@@ -0,0 +1 @@
+CARD_TYPE_ID_PREFIX = {'26': 'troop', '27': 'building', '28': 'spell'}
```

Comment:
Can you add a comment above this variable explaining what the numbers represent in case we look back at this a few months later and are like....wtf....

Reply:
done